docs/variants.rst

```diff
@@ -86,26 +101,138 @@ The option is itself a map with the following keys:
 ``short``
     A short human-readable string to describe the option.
 
-``long``
-    A lengthier human-readable string to describe the option. This value is
-    optional.
+``long`` (Optional)
+    A lengthier human-readable string to describe the option.
 
-``buildType``
+``buildType`` (Optional)
     An optional string to set for ``CMAKE_BUILD_TYPE`` when the option is
     active.
 
-``linkage``
+``linkage`` (Optional)
     Either ``static`` or ``shared``. Sets the value of
     ``CMAKE_BUILD_SHARED_LIBS``. This value is optional.
 
-``settings``
+``settings`` (Optional)
     A map of arbitrary CMake cache options to pass via the CMake command line
     with ``-D``. Similar to the ``cmake.configureSettings`` in
     ``settings.json``.
 
-``env``
+``env`` (Optional)
     A map of key-value string pairs specifying additional environment variables
     to set during CMake *configure* (not build). These environment variables
     take precedence over environment variables from ``settings.json``, the
     currently set :ref:`kit <kits>`, and environment variables set by the
-    system.
\ No newline at end of file
+    system.
+
+How Variants Are Applied
+========================
+
+A variant is a specific combination of one *option* from each of the defined
+*settings*. When CMake Tools executes the configure step, it will use the
+values from the currently active variant to determine the values to pass to the
+CMake process:
+
+#. Properties from all active options are merged. For ``env`` and ``settings``,
+   the objects themselves are merged. The merge order is unspecified, so
+   conflicting properties in options will result in unspecified behavior.
+#. All ``settings`` from the chosen options are passed as ``-D`` arguments to
+   the CMake process.
+#. The ``buildType`` is used for ``CMAKE_BUILD_TYPE``, the ``--config``
+   parameter to the build (For multi-conf generators), and for the CTest
```

Comment:
`parameter to the build (For multi-conf generators)` -> `parameter for the build (For multi-conf generators)`